DEV_GUIDE.md

```diff
@@ -360,26 +362,25 @@ First of all, the code must be compiled and the distribution artifacts created:
 mvn clean install -Dquick -Pdist
 ```
 
-If the tests are going to be run against local changes, use the [deploy-image.sh](./scripts/build-image.sh)
-describe [above](#building-and-pushing-a-kroxylicious-container-image) to create a test image.
-
-Start minikube:
+Start Minikube:
 ```shell
 minikube start
```

Comment:
```suggestion
minikube start --cpus 4
```
Do we need to use at least 4 cpus right?

Reply:
That's what I use, yes